backend/main/utils/dto.py

```diff
@@ -0,0 +1,34 @@
+class AnnouncementDTO:
```

Comment:
Suggestion, decorate these and make them `dataclass` for these. It'll automatically generate some boilerplate code like `__init___` and `__repr__` which would help us down the road if we ever need to troubleshoot. You would also be able to describe these DTOs with the typing information.

Reply:
done.